Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -1392,7 +1392,7 @@ def search_all_mailboxes():
             search_all_mailboxes()
 
 
-def search_command(mailbox: str = None, only_return_account_names: bool = False) -> dict[str, Any] | None:
+def search_command(mailbox: str = None, only_return_account_names: bool = False, first_time: bool = True) -> dict[str, Any] | None:
```

Comment:
to reduce confusion, check whether the following is possible. This will require anyone using to explicitly write `first_time=...

```suggestion
def search_command(mailbox: str = None, only_return_account_names: bool = False, *, first_time: bool = True) -> dict[str, Any] | None:
```